<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenuForm</class>
 <widget class="QWidget" name="MainMenuForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Bangers</family>
    <pointsize>64</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>471</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bauhaus 93</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font-size: 160px;
	background-color: transparent;
	color: #ffffff;
}</string>
   </property>
   <property name="text">
    <string>Ronin</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="gameButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>460</y>
     <width>261</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cascadia Mono SemiBold</family>
     <pointsize>-1</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font-size: 72px;
	border: none;
	background-color: transparent;
	color: #1c5ad6;
}</string>
   </property>
   <property name="text">
    <string>Game!</string>
   </property>
  </widget>
  <widget class="IconButton" name="historyIcon">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>570</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: none;
	background-color: transparent;
	border-image: url(:/icons/scroll.svg);
	border-width: 0px;
	background-position: center;
	background-repeat: no-repeat;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="IconButton" name="settingsIcon">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>570</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: none;
	background-color: transparent;
	border-image: url(:/icons/gear.svg);
	border-width: 0px;
	background-position: center;
	background-repeat: no-repeat;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="IconButton" name="exitIcon">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>570</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: none;
	background-color: transparent;
	border-image: url(:/icons/x.svg);
	border-width: 0px;
	background-position: center;
	background-repeat: no-repeat;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="backgroundLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <zorder>backgroundLabel</zorder>
  <zorder>exitIcon</zorder>
  <zorder>gameButton</zorder>
  <zorder>historyIcon</zorder>
  <zorder>settingsIcon</zorder>
  <zorder>title</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IconButton</class>
   <extends>QPushButton</extends>
   <header location="global">components/iconbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
